index.bs

```diff
@@ -259,6 +261,38 @@ the user agent SHOULD run these steps:
 2. If |video| is {{pictureInPictureElement}}, run the <a>exit
     Picture-in-Picture algorithm</a>.
 
+## Auto Picture-in-Picture ## {#auto-pip}
+
+Some pages may want to automatically enter and leave Picture-in-Picture for
+a video element; for example, video meetings web apps would benefit from
+some automatic Picture-in-Picture behavior when user switches back and forth
+between web app and other applications/tabs.
+To support these use cases, a new {{autoPictureInPicture}} attribute is
+added to the list of content attributes for video elements.
+
+A corresponding {{autoPictureInPicture}} IDL attribute which reflects the
+value of element’s {{autoPictureInPicture}} content attribute is added to
+the {{HTMLVideoElement}} interface. The {{autoPictureInPicture}} IDL attribute
+MUST <a>reflect</a> the content attribute of the same name.
+
+The <dfn>autoPictureInPictureElement</dfn> is the video element whose
+{{autoPictureInPicture}} attribute was set most recently.
```

Comment:
What happens if this element loses the attribute?

Reply:
Nothing.
Do you think something should happen?

Reply:
I guess "was set most recently" may be too vague. If I do: `video.setAttribute('autopictureinpicture', ''); videos.setAttribute('autopictureinpicture', undefined);`, `video` had the attribute set most recently but the page clearly said that it did not want to have it go in auto-pip anymore.

Reply:
Would it be better?

`the video element whose {{autoPictureInPicture}} content attribute was set most recently.`.

See https://html.spec.whatwg.org/multipage/common-dom-interfaces.html#reflect

Reply:
How about https://github.com/WICG/picture-in-picture/pull/112/commits/b12c3172213e67b7dab8e7b88b09974671db2b42